content/python/concepts/built-in-functions/terms/exec/exec.md

```diff
@@ -0,0 +1,100 @@
+--
+Title: 'exec()'
+Description: 'Executes a code object or string containing python code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The **`exec()`** function executes a code object or string containing python code.
+
+## Syntax
+
+```pseudo
+exec(object, globals=None, locals=None)
+```
+
+* `object` : String or code object
+* `globals`: (optional) A dictionary containing global variables. If not specified, defaults to `None`. 
+* `locals`: (optional) A dictionary containing global variables. If not specified, defaults to `None`.
+
+The `exec()` function returns `None`.
+
+## Example 1
+
+This example uses `exec()` to parse and execute Python code contained in the string `code`:
+
+```py
+code = 'print("Hello, Codecademy!")'
+
+exec(code)
+```
+
+## Example 2
+
+This example uses `exec()` to execute python code from a file `code.txt`, which contains Python commands:
+
+```py
+#code.txt
+import datetime
+
+current_time = datetime.datetime.now()
+
+print(current_time)
+```
+
+The content of the file `code.txt` is read until the end of the file (EOF) into a string. The content (Python commands) is executed by `exec()`.
+
+```py
+with open('code.txt','r') as file:
+  code = file.read()
+
+exec(code)
+```
+
+## Example 3
```

Comment:
I'd like you to swap this codebyte example with example 4 so this comes at the end. Then change the header to read `## Codebyte Example"